### DISCUSSION TITLE
How can I prevent user from changing their email?

### DISCUSSION THREAD

**Discussion by ser** [goauthentik/authentik#9914] [Q&A] [answered]
How can I prevent user from changing their email address and allow admins only to change them?

**Comment by ser** [comment, marked as answer]
OK, it can be done via '''/if/admin/#/admin/settings''' - my misunderstaning came from the fact that user UI still allows changing it and it's disallowed only after change attempt. IMHO the user interface should not allow the change at all, for example the form should be grayed out.

**Comment by Mrs-Feathers** [comment]
<img width="1264" height="874" alt="Screenshot 2025-09-11 at 4 29 08 PM" src="https://github.com/user-attachments/assets/978ed85d-0136-4563-8842-f5808f265557" />

theres nothing there

**Reply by dewi-tik** [reply]
You should be seeing this: 
<img width="2220" height="1542" alt="image" src="https://github.com/user-attachments/assets/177001b4-cff3-4170-bb59-bc862dee8ed6" />

Are you seeing any errors in your browser that would prevent the page from loading?
